src/templates/resources/footer.json

```
@@ -773,7 +773,7 @@
                 "text": "Restaurant sign up"
             },
             {
-                "url": "https://couriers.just-eat.co.uk/application",
+                "url": "https://just-eat.co.uk/info/delivering-with-just-eat",
```

```suggestion
                "url": "/info/delivering-with-just-eat",
```
minor. but just to be consistent

---

We have a mix of both variant through the file actually, so I don't think it will make any difference